main.py

```diff
@@ -227,7 +227,7 @@ def create_clan_members_ranked_embed(clan_info, clan_members_ranked, card_type_a
                 idx + 1, mc[14], mc[13], mc[12], mc[11], member['name']
```

Comment:
Lets group the lvl13/14 cards together too. i.e. in the line above this one. Change it to
```python
rank_text_row = '` {:2d} ` | ` ({:2d}, {:2d}), {:2d}, {:2d} ` | **{}**'.format(
```